packages/eslint-plugin-pf-codemods/src/rules/v6/userFeedbackWarnChanges/user-feedback-warn-changes.test.ts

```diff
@@ -0,0 +1,36 @@
+const ruleTester = require("../../ruletester");
+import * as rule from "./user-feedback-warn-changes";
+
+ruleTester.run("user-feedback-warn-changes", rule, {
+  valid: [
+    {
+      code: `<FeedbackModal />`,
+    },
+    {
+      code: `import { FeedbackModal } from '@patternfly/some-other-package';`,
+    }
+  ],
+  invalid: [
+    {
+      code: `import { FeedbackModal } from '@patternfly/react-user-feedback';`,
+      output: `import { FeedbackModal } from '@patternfly/react-user-feedback';`,
+      errors: [
+        {
+          message: `FeedbackModal no longer internally references a scss stylesheet. You may have to import "Feedback.css" located in the dist "@patternfly/react-user-feedback/dist/esm/Feedback/Feedback.css" to maintain styling on FeedbackModal.`,
+          type: "ImportDeclaration",
+        },
+      ],
+    },
+    // Test that a warning only gets flagged once
+    {
+      code: `import { FeedbackModal } from '@patternfly/react-user-feedback';`,
+      output: `import { FeedbackModal } from '@patternfly/react-user-feedback';`,
+      errors: [
+        {
+          message: `FeedbackModal no longer internally references a scss stylesheet. You may have to import "Feedback.css" located in the dist "@patternfly/react-user-feedback/dist/esm/Feedback/Feedback.css" to maintain styling on FeedbackModal.`,
+          type: "ImportDeclaration",
+        },
+      ],
+    },
```

Comment:
I'm a bit confused, what exactly is this second test testing that the first one isn't? They look the same?

Reply:
When I checked another codemod I saw a similar setup, but I'll remove it. I'm not sure how it's testing the warning is only flagged a single time, looking at it again.